website/pages/docs/k8s/service-sync.mdx

```diff
@@ -125,12 +125,12 @@ node that has the representative pod running. While Kubernetes configures
 a static port on all nodes in the cluster, this limits the number of service
 instances to be equal to the nodes running the target pods.
 
-The service instances will be registered to the Kubernetes node name
-that each instance lives on. This is guaranteed unique by Kubernetes. An
-existing node entry will be used if it is already part of the Consul
-cluster (for example if you're running a client agent on all Kubernetes
-nodes). This allows the normal agent health checks for that node to continue
-working.
+The service instance's IP will be set to the node IP that each pod is running on.
```

Comment:
This seems to rephrase what the first sentence in the previous paragraph is saying. Should we also mention that we register things to a synthetic node instead of the kubernetes node?

Reply:
I've rephrased this and added something above about the synthetic node, lmk what you think.

Reply:
Looks great!